exampleCourse/questions/demo/overlayDropdown/question.html

```diff
@@ -4,7 +4,7 @@
         This question combines a static image to serve as a background along with the <code class="user-output">pl-dropdown</code> element.
     </i>
 </p>
-<p>
+<p id="question-text">
```

Comment:
This ID is now unnecessary, right?

```suggestion
<p>
```

Reply:
[3f527d0](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/3f527d0bd67b8c09d9e8416a41c6855d08e6f262)